api/services/elastigroup/azure/scaleSetVms/schemas/lowPriorityVMsEGCreation.yaml

```diff
@@ -0,0 +1,1202 @@
+type: "object"
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master